src/llvm/lib_llvm.cr

```diff
@@ -10,7 +10,7 @@ end
   {% end %}
   @[Link(ldflags: {{"`#{LibLLVM::LLVM_CONFIG} --libs --system-libs --ldflags#{" --link-static".id if flag?(:static)}#{" 2> /dev/null".id unless flag?(:win32)}`"}})]
   lib LibLLVM
-    VERSION = {{`#{LibLLVM::LLVM_CONFIG} --version`.chomp.stringify}}
+    VERSION = {{`#{LibLLVM::LLVM_CONFIG} --version`.chomp.stringify.gsub(/git/,"")}}
```

Comment:
For future reviewers: Note there is no equivalent to `String#rchop` for `StringLiteral` which might've been nice for this.